### DISCUSSION TITLE
v1.1.1

### DISCUSSION THREAD

**Discussion by dirkgroot** [avisi-cloud/structurizr-site-generatr#226] [Announcements]
### Bugfixes

* Fixed SiteBranding CSS bug for generated sites by @qtzar in https://github.com/avisi-cloud/structurizr-site-generatr/pull/220
* Prevent invalid links in diagrams  by @jp7677 in https://github.com/avisi-cloud/structurizr-site-generatr/pull/223
* Clear search results before adding actual results by @jp7677 in https://github.com/avisi-cloud/structurizr-site-generatr/pull/224

### Dependency updates

* fix(deps): update dependency org.jetbrains.kotlinx:kotlinx-serialization-json to v1.5.1 by @renovate in https://github.com/avisi-cloud/structurizr-site-generatr/pull/225

**Full Changelog**: https://github.com/avisi-cloud/structurizr-site-generatr/compare/1.1.0...1.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/avisi-cloud/structurizr-site-generatr/releases/tag/1.1.1'>v1.1.1</a>.</em>

**Comment by jp7677** [comment]
Also credits and thanks to @drazzib for his work on https://github.com/avisi-cloud/structurizr-site-generatr/pull/223
